JavaScript/JavaScriptSDK.Tests/CheckinTests/appInsights.tests.ts

```diff
@@ -38,7 +38,6 @@ class AppInsightsTests extends TestClass {
     }
 
     public testCleanup() {
-        this.clock.reset();
```

Comment:
this.clock is undefined at this point. I think it's being cleared automatically by the sandbox. See: https://github.com/Microsoft/ApplicationInsights-JS/blob/b056eae7cc92c21424e41f90724bcc593621833a/JavaScript/JavaScriptSDK.Tests/TestFramework/TestClass.ts#L163